### PR TITLE
helix 24.03

### PR DESCRIPTION
**Pull request by dirksierd**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/h/helix.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 class Helix < Formula
   desc "Post-modern modal text editor"
   homepage "https://helix-editor.com"
-  url "https://github.com/helix-editor/helix/releases/download/23.10/helix-23.10-source.tar.xz"
-  sha256 "4e7bcac200b1a15bc9f196bdfd161e4e448dc670359349ae14c18ccc512153e8"
+  url "https://github.com/helix-editor/helix/releases/download/24.03/helix-24.03-source.tar.xz"
+  version "24.03"
```
Comment (chenrui333) [inline Formula/h/helix.rb:5]:
```suggestion
```

Comment (chenrui333) [reply Formula/h/helix.rb:5]: this is redundant

Comment (dirksierd) [reply Formula/h/helix.rb:5]: Well, that was my initial commit, but that led to a failing testcase. Homebrew presumed the version to be 24.3, not 24.03, so the regex-match failed.

Comment (chenrui333) [reply Formula/h/helix.rb:5]: looks like upstream issue with the version identifier

Comment (branchvincent) [reply Formula/h/helix.rb:5]: Let's replace the version test, it's not really valuable anyway

**Review by p-linnane — APPROVED** [review]

**Pull request merged**